MANIFEST.in

```diff
@@ -1,3 +1,11 @@
-include gfootball/build_game_engine.sh
-recursive-include third_party/gfootball_engine/ *
-recursive-include third_party/fonts/ *
+include gfootball/build_game_engine.*
+recursive-include gfootball/doc *
+recursive-include gfootball/eval_server/proto *
+
+recursive-include third_party/gfootball_engine *
+recursive-exclude third_party/gfootball_engine *.a libgame.so cmake_install.cmake Makefile
+prune third_party/gfootball_engine/CMakeFiles
+prune third_party/gfootball_engine/cmake-build-debug
+prune third_party/gfootball_engine/build*
+prune third_party/gfootball_engine/.vs
+prune third_party/gfootball_engine/.idea
```

Comment:
I reorganized Manifest.in to include non-python files in gfootball and exclude generated files in gfootball_engine.